SSI.php

```diff
@@ -25,9 +25,6 @@
 global $smcFunc, $ssi_db_user, $scripturl, $ssi_db_passwd, $db_passwd, $cachedir;
 
 // Remember the current configuration so it can be set back.
-$ssi_magic_quotes_runtime = function_exists('get_magic_quotes_gpc') && get_magic_quotes_runtime();
-if (function_exists('set_magic_quotes_runtime'))
-	@set_magic_quotes_runtime(0);
```

Comment:
If 2.0.14 raised the minimum to 5.4 I don't think SMF 2.1 works with anything lower than that anyway. Especially since some code was ported from 2.1 to 2.0.14 (including the DB class causing the issue).